tests/test_image_handling.py

```diff
@@ -25,3 +41,32 @@ def test_scale_image(self):
         assert scale_image(self.img_96x85, False, "50,", "image/jpeg") == self.img_50x44
         assert scale_image(self.img_96x85, False, "60,44", "image/jpeg") == self.img_49x44
         assert scale_image(self.img_96x85, False, ",44", "image/jpeg") == self.img_49x44
+
+    #TODO: Specifiy exception
+    def test_parse_region_string(self):
+        with pytest.raises(Exception):
+            parse_region_string("x,y,w,h")
+            parse_region_string("")
+            parse_region_string(",,,")
```

Comment:
Member
@[kramer65](https://github.com/kramer65) kramer65 [Oct 30, 2023](https://github.com/Amsterdam/iiif-auth-proxy/pull/174/commits/e48935bcf695406652016d003105441ba379925a#r1376230216)

Hier test je of één van die drie een random exception geeft. Als je dat bij alledrie verwacht, dan moet je die wel apart testen. Je kan er een loop van maken.
Verder test je nu alleen of er "een exception" wordt geraised, niet wélke exception. Zie regel twee van de zen of python.